daphne/src/messages/mod.rs

```diff
@@ -361,13 +359,13 @@ pub struct AggregateInitializeReq {
 }
 
 impl ParameterizedEncode<DapVersion> for AggregateInitializeReq {
-    fn encode_with_param(&self, encoding_parameter: &DapVersion, bytes: &mut Vec<u8>) {
+    fn encode_with_param(&self, version: &DapVersion, bytes: &mut Vec<u8>) {
         self.task_id.encode(bytes);
         self.agg_job_id.encode(bytes);
-        match encoding_parameter {
+        match version {
             DapVersion::Draft02 => encode_u16_bytes(bytes, &self.agg_param),
             DapVersion::Draft03 => encode_u32_bytes(bytes, &self.agg_param),
-            _ => panic!("unimplemented DapVersion"),
+            _ => unreachable!(),
```

Comment:
I believe you can still output a message, which might be helpful if in fact we manage to panic here.
```suggestion
            _ => unreachable!("unimplemented version of DAP"),
```